pages/src/common/buttons.less

```diff
@@ -34,58 +36,72 @@
   &:focus-visible {
     outline: 2px dashed #1fc2ff;
   }
-  &:hover {
-    color: #212529;
-    text-decoration: none;
-  }
   &:disabled {
     opacity: 0.65;
   }
 
+  color: var(--btn-color);
+  border: 2px solid var(--btn-color);
+  border-radius: 0.75rem;
```

Comment:
Do we need consider adding `number-leading-zero` rule to stylelint config?
I have looked through our less files we use both options with leading zero and without leading zero.

Reply:
Issue created